docs/whatsnew/3.1.rst

```diff
@@ -520,5 +522,105 @@ To see a detailed list of all changes in version v3.1, including changes in
 API, please see the :ref:`changelog`.
 
 
-Renamed/removed functionality
-=============================
+Contributors to the v3.1 release
+================================
+
+.. hlist::
```

Comment:
Need to check rendered doc in CircleCI to make sure it renders as expected, as the GitHub preview does not.

Reply:
Absolutely. Otherwise I works for me ™️ 😄

Reply:
I double checked this and it rendered fine locally!